### ISSUE TITLE
Better Wording for Organization Management Page

### ISSUE THREAD

**Issue by lomky** [department-of-veterans-affairs/caseflow#11613]
Right now the organization management page titles the page "[Team Name] team", leading to pages titled "Privacy Team team" and "Case Movement Team team". Let's reword it to something more akin to "[Team name] management page"

Code to update is here: https://github.com/department-of-veterans-affairs/caseflow/blob/faec57a739aceed4a1d20f1ccd49c55f2c3d7aec/client/app/queue/OrganizationUsers.jsx#L209

Should be a real easy one!

[Related Slack conversation](https://dsva.slack.com/archives/CJL810329/p1564585242073500) for reference.

**Comment by lowellrex** [comment]
_[What is this graph?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```
1 | |||||||||
2 | 
3 | 
5 | 
8 | 
```
